app/views/visualizations/amr_heatmap.html.erb

```diff
@@ -0,0 +1,12 @@
+<div id="AMRHeatmapView_page">
```

Comment:
I would remove this file, since it's not currently being used. Don't include files intended for future PRs.

Reply:
Removed! Sorry about that, my oversight.